corehq/util/sentry.py

```diff
@@ -38,6 +51,13 @@ def should_capture(self, exc_info):
         if not super(HQSentryClient, self).should_capture(exc_info):
             return False
 
+        try:
+            value = str(ex_value)
+            if looks_sensitive(value):
+                return False
+        except Exception:
+            # don't fail if something unexpected happened
```

Comment:
This always makes me nervous. The above probably won't raise an exception, but if something unexpected _does_ happen—don't you want it to leave a record somewhere and/or email you? Maybe a soft_assert would be good here.

Reply:
(oh, just saw your last comment.)